src/main/resources/templates/error.html

```diff
@@ -0,0 +1,15 @@
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta charset="UTF-8"/>
+  <title>Not Found</title>
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+  <link href="https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz" rel="stylesheet" type="text/css">
+  <link href="https://faforever.com/styles/css/site.min.css" rel="stylesheet" type="text/css"/>
+  <link rel="stylesheet" th:href="@{/css/style.css}" type="text/css"/>
```

Comment:
BTW: How does `@{/css/style.css}` works?

Reply:
I think it's some thymeleaf stuff